### PR TITLE
Md sa2 23 23 - 05 and 06 HW

### PR DESCRIPTION
**Pull request by julietredk**

### CODE REVIEW

File: Yulia_Redkova/06.Ansible_workshop/playbook_nginx.yaml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        state: directory
+        mode: '0755'
+    - name: Create a directory if it does not exist
+      ansible.builtin.file:
+        path: /var/www/julia-site2.by/public_html/
+        state: directory
+        mode: '0755'
+
+    - name: Create html file for virtual host1
+      ansible.builtin.template:
```
Comment (pluhin) [inline Yulia_Redkova/06.Ansible_workshop/playbook_nginx.yaml:22]: Do not duplicate code, especially when you use template modue

File: Yulia_Redkova/06.Ansible_workshop/playbook_nginx.yaml [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        remote_src: yes
+
+    - name: Copy a "/etc/nginx/sites-available/default" file on the remote machine for site2
+      ansible.builtin.copy:
+        src: /etc/nginx/sites-available/default
+        dest: /etc/nginx/sites-available/julia-site2.by
+        remote_src: yes
+
+    - name: editing /etc/nginx/sites-available/julia-site1.by
+      ansible.builtin.replace:
```
Comment (pluhin) [inline Yulia_Redkova/06.Ansible_workshop/playbook_nginx.yaml:49]: Please use template module here, you can generate config files

**Review by pluhin — CHANGES_REQUESTED** [review]

**Comment by julietredk** [comment]
fixed.

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**